src/components/atom/main/sidebar/SocialIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { SocialIconImage } from "./SocialIconStyles";
+
+interface Props {
+  src: string;
+  alt: string;
+}
+
+const SocialIcon: React.FunctionComponent<Props> = ({ src, alt }) => {
+  return <SocialIconImage src={src} alt={alt} />;
+};
```

Comment:
위 의견과 같이 map 함수 없이 작성하려면 다음과 같이 작성해야 할 것 같은데 참고로 봐주세요!
```suggestion
const SocialIcon = ({ src, alt, href }: Props) => {
  return (
    <a href={href} target="_blank" rel="noopener noreferrer">
      <SocialIconImage src={src} alt={alt} />
    </a>
  );
};
```

Reply:
위의 의견에 따라 map함수 없이 작성하는게 보기 간편한 것 같네요 수정해 놓겠습니다